.goreleaser.yml

```diff
@@ -263,7 +252,6 @@ dockers:
     extra_files:
       - release_deps/VERSION.txt
       - release_deps/plugins
-      - config/logging.stdout.yaml
```

Comment:
Will containers continue to log to stdout / error?

Reply:
Unsure about containers. All I know is the supervisor starts the agent as it's own subprocess and directs output from stdout to a file called `agent.log`. I'm not sure what that means for containers...

[Relevant code](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/opampsupervisor/supervisor/commander/commander.go#L74-L91)

Reply:
I think this should be a blocker that we resolve before cutting over to v2. Logging to a file within the container will be a no go.

To be clear, it does not need to block this PR, but we do need to track this as an issue.